src/esbmc/options.cpp

```diff
@@ -200,7 +200,8 @@ const struct group_opt_templ all_cmd_options[] = {
     {"no-slice", NULL, "do not remove unused equations"},
     {"multi-fail-fast",
      boost::program_options::value<int>()->value_name("n"),
-     "stops after first n VCC violation found in multi property mode"},
+     "stops after first n VCC violation found in multi property mode."
+     "(not compatible with incremental bmc strategies)"},
```

Comment:
You should move this from the option and check it at the code level. If it is a hard requirement you should crash or report with an warning if its just buggy.

Reply:
Will do